src/components/implementation/no_interface/new_boot/boot_deps.h

```diff
@@ -28,24 +38,24 @@ boot_comp_map_memory(struct cobj_header *h, spdid_t spdid, pgtblcap_t pt)
 	int flag;
 	vaddr_t dest_daddr, prev_map = 0;
 	int tot = 0, n_pte = 1;	
-	
 	struct cobj_sect *sect = cobj_sect_get(h, 0);
 	
-	/*Expand Page table, could do this faster*/
-	for(j = 0; j < (int)h->nsect; j++){
+	/* Expand Page table, could do this faster */
```

Comment:
If we put this entire function in here, that means that we have to replicate all of it in each booter.  I thought we could share *some* of its logic.  I might be misremembering (and don't have time to check right now).

I would hope that we could use the previous booter with minimal modifications, and use the call-outs to boot_deps that it used.  Perhaps not, but I'd like to understand why or why not.